dev/com.ibm.ws.security.audit.reader.auditreader/bnd.bnd

```diff
@@ -13,6 +13,8 @@ bVersion=1.0
 Bundle-Name: WebSphere Audit Reader Utility
 Bundle-SymbolicName: com.ibm.ws.security.audit.reader.auditreader
 
+jakartaeeMe: true
```

Comment:
auditreader doesn't appear to have any Java EE dependencies unless I am missing something.  Why does it need to be jakarta-ized?